test/service-worker-manager.test.ts

```
@@ -0,0 +1,135 @@
+            if (!cb) {
+                listeners.delete(event);
+                return;
+            }
+            const cbs = listeners.get(event);
+            if (!cbs) return;
+            const idx = cbs.indexOf(cb);
+            if (idx !== -1) cbs.splice(idx, 1);
+            if (cbs.length === 0) listeners.delete(event);
+        },
+        dispatch: (event: string, evt: Event = new Event(event)) => {
+            listeners.get(event)?.forEach((cb) => cb(evt));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)